### PR TITLE
Update CHANGELOG.md for 4.66.0

### PR DESCRIPTION
**Pull request by trwalke**
Fixes #

**Changes proposed in this request**

**Testing**

**Performance impact**

**Documentation**
- [ ] All relevant documentation is updated.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+4.66.0
+==========
+### New Features
+- Enabled extended functionality in the MSAL authentication flow. See [Issue #4956](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4956)
```
Comment (gladjohn) [inline CHANGELOG.md:4]:
```suggestion
- Enabled extended functionality in the MSAL authentication flow through the implementation of `IAuthenticationOperation`. This allows for custom extensions in the authentication flow by providing an authentication operation interface. See [Issue #4956](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4956)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+- MSAL.NET will now opt-in to regional when env variable MSAL_FORCE_REGION is set . See [Issue #4930](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4930)
```
Comment (gladjohn) [inline CHANGELOG.md:5]:
```suggestion
- MSAL.NET will now opt-in to regional ESTS when the environment variable MSAL_FORCE_REGION is set `(e.g., MSAL_FORCE_REGION=eastus)`. This automatically directs MSAL to use the specified region for token acquisition. If the region is explicitly set to `"DisableMsalForceRegion"`, MSAL will not opt-in to regional ESTS. Use of the `WithAzureRegion(xyz)` API takes precedence over the environment variable. See [Issue #4930](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4930)
```

**Review by gladjohn — APPROVED** [review]

**Comment by gladjohn** [comment]
PR builds are not running anymore?

**Comment by trwalke** [comment]
> PR builds are not running anymore?

Interesting, I guess not. Need to loo at the trigger settings

**Review by gladjohn — APPROVED** [review]

**Comment by Monster43-beep** [comment]
Luckily

**Review by Monster43-beep — COMMENTED** [review]
- - [ ] - ****__****bfe@hhtzyoxgi gi8(,)9 goztxgi(8th it to to you to go to

**Pull request merged**